examples-be/src/main.ts

```diff
@@ -56,22 +45,15 @@ class TokenService {
    * Getting installation-based access token: https://developers.outreach.io/api/app-token/#get-s2s-access-token
    * @param installationId
    */
-  public async getInstallationBasedToken(
```

Comment:
@sheff3rd this file is full of formatting changes, please make sure that prettier setup is correct.

Reply:
I think a better way of handling the prettier fixes would be a separate PR with prettier fixed and enforced as a build check. @palootcenas-outreach wdyt?

Reply:
I agree

Reply:
I think it's OK to merge it in the current state but for the future it's a good practice to split refactoring/formatting and functional changes into separate PRs. Makes it easier to review, easier to track changes or manage possible reverts.